asminc/accelerator.inc

```diff
@@ -0,0 +1,34 @@
+;
+; Accelerator definitions.
+;
+
+; ---------------------------------------------------------------------------
+; Speed definitions for all accelerator, to be used as input for the 'set'
+; functions.
+
+
+SPEED_SLOW := $00
```

Comment:
":=" is intended for addresses while "=" is intended for general constants. See http://cc65.github.io/doc/ca65.html#ss6.1

Reply:
Any comments on the design are welcome of course, but besides the change needed for the constant definition it is ready for merging, other accelerator code will follow after it gets merged, including for other platforms like apple2 and pet.